boards/shields/xenvm_dom0/doc/index.rst

```diff
@@ -0,0 +1,29 @@
+.. _xenvm_dom0:
+
+Xen VM Dom0: universal shield for XEN VM Dom-0
+#####################################################################
+
+Overview
+********
+
+This is a virtual shield that allows you to use any board config as a Xen
+initial domain.
```

Comment:
Strange wording. You can use "board config" as "initial domain". "  How about "This virtual shield allows user to build Zephyr as a Xen initial domain (Dom0). This feature is implemented as shield to allow support for any compatible platform".